marketplace-ui/src/app/modules/product/product-detail/product-detail-version-action/product-detail-version-action.component.ts

```diff
@@ -41,6 +45,15 @@ export class ProductDetailVersionActionComponent implements AfterViewInit {
     }));
   });
 
+  selectedVersionInDesigner = model<string>('');
```

Comment:
let us the same property with website versions, since they exactly handle the same thing and same logic